.pre-commit-config.yaml

```diff
@@ -1,3 +1,44 @@
+# This is the configuration file for the pre-commit framework, a simple way
+# to manage, install and run git hooks to catch common problems early on.
+# See https://pre-commit.com/ for details.
+#
+# This adds the following hooks:
+#
+# - check-case-conflict
+# - check-byte-order-marker
+# - check-yaml
+# - check-json
+# - check-xml
+# - mixed-line-ending
+# - end-of-file-fixer
+# - trailing-whitespace
+# - no-commit-to-branch
+# - eslint
+# - clang-format
+# - qsscheck
+#
+# If you have python and python-pip installed, just run:
```

Comment:
Please add also a hint that Python 3.7 is a minimum requirement

Reply:
Done.